tfx/components/base/base_executor_test.py

```diff
@@ -37,21 +44,130 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
     pass
 
 
+class _BeamTestExecutor(base_executor.BeamExecutor):
+  """Fake BeamExecutor for testing purpose only."""
+
+  def beam_io_signature(self, input_dict: Dict[Text, List[types.Artifact]],
+                        output_dict: Dict[Text, List[types.Artifact]],
+                        exec_properties: Dict[Text, Any]
+                        ) -> List[Dict[Text, Any]]:
+    input_signature = {_EXAMPLES: io.TextIOBase}
```

Comment:
Should this just be `Text`?

Reply:
You're right, I changed the types to `Text` and `int`